README.md

```diff
@@ -139,6 +141,16 @@ host: localhost
 #profile_host: 127.0.0.1
 #profile_port: 7070
 
+# If specified, the HTTP read timeout for a client request in seconds.
+# (applies to the frontend not the proxy backend)
+# Time units can be one of: "s", "m", "h".
+#http_read_timeout: 5s
```

Comment:
How did you arrive at these example values (5 and 10s)? Is this an upper limit on the expected time for the server to receive an upload, and to write a response back?

Reply:
These limits are just examples, It varies depending on blob sizes, etc. there is no one-fit-for-all :/

Reply:
I was thinking we could recommend something like double the maximum time expected for clients to upload/download items from the cache- ie a rough rule that can be used as a starting point. What do you think?

Reply:
sounds good ! @mostynb can you merge and update the docs then ?

Reply:
Sure.